### PR TITLE
Code style fixes

### PR DESCRIPTION
**Pull request by webimpress**
**Changes:**
- PHPDocs consistency (added missing variables, return statements, throws tags)
- CS fixes (indents, trailing commas)
- Simplified conditionals and reduced redundant braces
- Removed unused variables
- `::class` notation
- Missing license headers
- Optimized imports (removed redundancies, alphabetical order)

**TODO / Quesions:**
- [ ] `setRequest` method in `AuthorizationController` should be removed or at least marked as `@deprecated`
- [ ] `VersioningModelFactoryFactory` should be marked as `@deprecated` as long as `VersioningModelFactory` is deprecated
- [ ] upgrade script should leave one empty line at the end of `modules.config.php` and `composer.json`
- [ ] upgrade script should remove `ZFTool` module from `development.config.php` and `development.config.php.dist`
- [ ] HHVM tests need to be checked - many of them are failing
- [ ] remove `markTestSkipped` in some mongo tests (fix if needed) - all test should run on travis
- [ ] check tests `sh: 1: tree: not found` in https://travis-ci.org/zfcampus/zf-apigility-admin/jobs/150746938#L415

### CODE REVIEW

File: src/InputFilter/Authentication/DigestInputFilter.php [hunk 1]
```diff
@@ -53,7 +53,7 @@ public function init()
         $this->add([
             'name' => 'nonce_timeout',
             'validators' => [
-                ['name' => 'Zend\Validator\Digits']
+                ['name' => \Zend\Validator\Digits::class],
```
Comment (weierophinney) [inline src/InputFilter/Authentication/DigestInputFilter.php]: We should likely import this class. I can do that on merge.

File: src/Model/AbstractAutodiscoveryModel.php [hunk 2]
```diff
@@ -28,18 +32,18 @@
      */
     protected $validators = [
         'text' => [
-            'name' => 'Zend\Validator\StringLength',
+            'name' => \Zend\Validator\StringLength::class,
```
Comment (weierophinney) [inline src/Model/AbstractAutodiscoveryModel.php]: Should import all of these. I'll do that on merge.

File: src/Model/AuthenticationModel.php [hunk 3]
```diff
@@ -6,19 +6,18 @@
...
 use ZF\Rest\Exception\CreationException;
-use ZF\Apigility\Admin\InputFilter\Authentication;
 
 class AuthenticationModel
 {
-
-    const ADAPTER_HTTP   = 'ZF\\MvcAuth\\Authentication\\HttpAdapter';
-    const ADAPTER_OAUTH2 = 'ZF\\MvcAuth\\Authentication\\OAuth2Adapter';
+    const ADAPTER_HTTP   = \ZF\MvcAuth\Authentication\HttpAdapter::class;
+    const ADAPTER_OAUTH2 = \ZF\MvcAuth\Authentication\OAuth2Adapter::class;
```
Comment (weierophinney) [inline src/Model/AuthenticationModel.php]: Should import these classes.

File: src/Model/AuthenticationModel.php [hunk 4]
```diff
@@ -612,14 +613,14 @@ protected function patchOAuth2Config(AuthenticationEntity $entity, array $global
         switch ($entity->getDsnType()) {
             case AuthenticationEntity::DSN_MONGO:
                 $toSet = [
-                    'storage' => 'ZF\OAuth2\Adapter\MongoAdapter',
+                    'storage' => \ZF\OAuth2\Adapter\MongoAdapter::class,
```
Comment (weierophinney) [inline src/Model/AuthenticationModel.php]: Should import these.

File: src/Model/DbConnectedRestServiceEntity.php [hunk 5]
```diff
@@ -6,11 +6,13 @@
...
 namespace ZF\Apigility\Admin\Model;
 
+use ZF\Rest\Exception\CreationException;
+
 class DbConnectedRestServiceEntity extends RestServiceEntity
 {
     protected $adapterName;
 
-    protected $hydratorName = 'Zend\Hydrator\ArraySerializable';
+    protected $hydratorName = \Zend\Hydrator\ArraySerializable::class;
```
Comment (weierophinney) [inline src/Model/DbConnectedRestServiceEntity.php]: Import this.

File: src/Model/DoctrineAdapterModel.php [hunk 6]
```diff
@@ -80,8 +80,7 @@ public function remove($name)
     public function fetchAll()
     {
         $fromConfigFile = $this->localConfig->fetch(true);
-        if (isset($fromConfigFile['doctrine'])
-            && isset($fromConfigFile['doctrine']['connection'])
+        if (isset($fromConfigFile['doctrine']['connection'])
```
Comment (weierophinney) [inline src/Model/DoctrineAdapterModel.php]: Thanks! I thought I'd caught all of these yesterday!

**Comment by weierophinney** [comment]
> HHVM tests need to be checked - many of them are failing

This has always been true, to my knowledge. Primarily, it's around some discrepancies in the two engines around references and ArrayObject implementations.

The only way we'll resolve those is for somebody with a working HHVM setup to volunteer to test; we've never prioritized it on our team.

**Comment by webimpress** [comment]
@weierophinney thanks!

**Pull request merged**